circle.yml

```diff
@@ -0,0 +1,15 @@
+machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
+  services:
+    - docker
+
+dependencies:
+  override:
+    - docker info
+    - docker pull slicer/slicer-build-deps
+
+test:
+  override:
+    - "git diff --name-only master | grep -q SuperBuild > /dev/null || echo 'Notice: TravisCI does not build changes to Slicer dependencies'"
```

Comment:
Is this line right? It looks like `grep` exits with a zero status code on a match, so the warning won't be printed. Also, does this line need to explicitly call `exit` with a non-zero status code to cause the build to fail? And--nitpick--it should say `CircleCI` instead of `TravisCI`.